docs/README.md

```diff
@@ -26,9 +26,29 @@ ERROR:  While executing gem ... (Errno::EPERM)
     Operation not permitted - /usr/bin/listen
 ```
 
+If you are using Mac OSX 10.11+ (El Capitan), you will need to execute following command:
+```
+sudo gvim /Library/Ruby/Gems/2.0.0/gems/ffi-1.9.10/lib/ffi/library.rb
```

Comment:
vim/gvim may not have been installed.  How about "If you are using Mac OSX 10.11+ (El Capitan), you need to edit `/Library/Ruby/Gems/2.0.0/gems/ffi-1.9.10/lib/ffi/library.rb` with `sudo` user, ..."